app/src/main/java/de/tum/in/tumcampusapp/services/GcmReceiverService.java

```diff
@@ -97,8 +97,10 @@ public void onMessageReceived(RemoteMessage message) {
                     Utils.log(e);
                 }
 
-                //de.tum.in.tumcampusapp.managers.NotificationManager man = new de.tum.in.tumcampusapp.managers.NotificationManager(this);
-                //@todo save to our notificationmanager
+                //TODO
+                //de.tum.in.tumcampusapp.managers.NotificationManager notificationManager
+                //        = new de.tum.in.tumcampusapp.managers.NotificationManager(this);
+                //notificationManager.replaceInto(n);
```

Comment:
Is this still relevant?

Reply:
yes, but as I don't have the full GCMNotification but only Key/Value List, its kinda tricky to insert it. Will handle it in a separate PR in the next weeks.